torchchat/model.py

```diff
@@ -399,8 +397,9 @@ def reset_caches(self):
 
 
 class Transformer(nn.Module):
-    def __init__(self, config: TransformerArgs) -> None:
+    def __init__(self, config: Dict[str, Any]) -> None:
```

Comment:
Not a fan of this one, Transformer taking TransformerArgs is the most intuitive set up and matches the other classes

Reply:
SG. Bring it back